pkg/importer/vddk-datasource_amd64.go

```diff
@@ -717,7 +719,8 @@ func (sink *VDDKFileSink) ZeroRange(offset uint64, length uint32) error {
 	if sink.isBlock { // Try to punch a hole in block device destination
 		err = punch(offset, length)
 	} else {
-		info, err := sink.file.Stat()
```

Comment:
What is the error for this?

Reply:
The problem is in line 720. this line creates a shadow err variable. This variable is never checked and actually ignored.

https://github.com/kubevirt/containerized-data-importer/blob/f9c049f0e481d4697dfa9444db8aa64620a17e91/pkg/importer/vddk-datasource_amd64.go#L716-L730

The linter created two warnings for that:
```
pkg/importer/vddk-datasource_amd64.go:725:5: ineffectual assignment to err (ineffassign)
                                err = syscall.Ftruncate(int(sink.file.Fd()), int64(offset+uint64(length)))
                                ^
pkg/importer/vddk-datasource_amd64.go:727:5: ineffectual assignment to err (ineffassign)
                                err = punch(offset, length)
```